Clusteredverticalbar.js

```diff
@@ -0,0 +1,994 @@
+    }
+
+    chart.valuePosition = function (value) {
+        if (!arguments.length) {
+            return _valuePosition;
+        }
+        _valuePosition = value;
+        return chart;
+    }
+
+    chart.sort = function (value) {
+        if (!arguments.length) {
+            return _sort;
+        }
+        _sort = value;
+        return chart;
+    }
+
+    chart.filterData = function (value) {
+        if (!arguments.length) {
+            return filterData;
+        }
+        filterData = value;
+        return chart;
+    }
+
+    chart.tooltip = function (value) {
+        if (!arguments.length) {
+            return _tooltip;
+        }
+        _tooltip = value;
+        return chart;
+    }
+
+    chart.showXaxis = function (value) {
+        if (!arguments.length) {
+            return _showXaxis;
+        }
+        _showXaxis = value;
+        return chart;
+    }
+
+    chart.showYaxis = function (value) {
+        if (!arguments.length) {
+            return _showYaxis;
+        }
+        _showYaxis = value;
+        return chart;
+    }
+
+    chart.showXaxisLabel = function (value) {
+        if (!arguments.length) {
+            return _showXaxisLabel;
+        }
+        _showXaxisLabel = value;
+        return chart;
+    }
+
+    chart.showYaxisLabel = function (value) {
+        if (!arguments.length) {
+            return _showYaxisLabel;
+        }
+        _showYaxisLabel = value;
+        return chart;
+    }
+
+    chart.xAxisColor = function (value) {
+        if (!arguments.length) {
+            return _xAxisColor;
+        }
+        _xAxisColor = value;
+        return chart;
+    }
+
+    chart.yAxisColor = function (value) {
+        if (!arguments.length) {
+            return _yAxisColor;
+        }
+        _yAxisColor = value;
+        return chart;
+    }
+
+    chart.showGrid = function (value) {
+        if (!arguments.length) {
+            return _tooltip;
+        }
+        _showGrid = value;
+        return chart;
+    }
+
+    chart.stacked = function (value) {
+        if (!arguments.length) {
+            return _tooltip;
+        }
+        _stacked = value;
+        return chart;
+    }
+
+    chart.displayName = function (value) {
+        if (!arguments.length) {
+            return _tooltip;
+        }
+        _displayName = value;
+        return chart;
+    }
+
+    chart.forEachMeasure = function (value) {
+        if (!arguments.length) {
+            return _measureProp;
+        }
+        _measureProp = value;
+        return chart;
+    }
+
+    chart.legendData = function (measureConfig, measureName) {
+        _legendData = {
+            measureConfig: measureConfig,
+            measureName: measureName
+        }
+        return _legendData;
+    }
+
+    chart.Local_data = function () {
+        if (!arguments.length) {
+            return _Local_data;
+        }
+        _Local_data = value;
+        return chart;
+    }
+
+
+    chart.showValues = function (value) {
```

Comment:
Show values can take single numerical input or an array of numbers input, where is the logic to handle both the cases. Please refer to KPI.js file for how this has been implemented and please do the same for other applicable accessor functions as well.